msg/LidarStatus.msg

```diff
@@ -0,0 +1,10 @@
+uint16 status
+
+uint16 NORMAL=0
+uint16 SETTING=1
+uint16 operating_mode
+
+uint16 area_number
+bool error_status
+uint16 error_code
+bool lockout_status
```

Comment:
Comments with documentation references for these fields, please.

Also, this message should probably have a header?